components/prism-core.js

```diff
@@ -460,8 +454,8 @@ function Token(type, content, alias, matchedStr, greedy) {
 }
 
 Token.stringify = function(o, language) {
-	if (typeof o == 'string') {
-		return o;
+	if (typeof o === 'string') {
+		return _.util.encode(o);
```

Comment:
```suggestion
		return o
			.replace(/&/g, '&amp;')
			.replace(/>/g, '&gt;')
			.replace(/</g, '&lt;')
			.replace(/"/g, '&quot;')
			.replace(/\u00a0/g, ' ');
```

Reply:
Unfortunately, we [can't do that](https://github.com/PrismJS/prism/blob/2f9c9261bc1454899266929711d5842a3675e467/components/prism-core.js#L246).

Replacing that line with `insertHighlightedCode(_.Token.stringify(env.code));` would work but it's not clear at all why a string has to be _stringified_, so I'd rather not do that.